server/app/query/view/[id]/page.tsx

```diff
@@ -86,18 +89,90 @@ export default function QueryPage({ params }: { params: { id: string } }) {
 
   return (
     <>
-      <div className="inline-flex items-center">
-        <h2 className="text-2xl font-bold leading-7 text-gray-900 dark:text-gray-100 sm:truncate sm:text-3xl sm:tracking-tight">
-          Query Details: {params.id}
-        </h2>
+      <div className="flex justify-between items-center">
+        <div>
+          <h2 className="text-2xl font-bold leading-7 text-gray-900 dark:text-gray-100 sm:truncate sm:text-3xl sm:tracking-tight">
+            Query Details
+          </h2>
+        </div>
+        <div>
+          <button
+            onClick={() => kill(IPARemoteServers)}
```

Comment:
I am not up to date with the best practices for developing UIs, is it the recommended way to execute actions from the HTML markup?

Reply:
yep! react is basically php but in JavaScript (not surprising since it came out of FB...)